release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+provides a visual representation of the applications within a project, their
+build status, and the components and services associated with them,  enabling
+easy interaction and monitoring. It incorporates Serverless capabilities
+(Technology Preview) and the ability to create workspaces to edit your
+application code using Eclipse Che.
+
+[id="ocp-4-2-prometheus-queries"]
+==== Prometheus queries
+
+You can now run Prometheus queries directly in the web console.
+
+[id="ocp-4-2-general-web-console-updates"]
+==== General web console updates
+
+* The dashboard is redesigned with more metrics.
+* *Catalog* is moved.
+* Status of projects is now moved to the *Workloads* menu.
+* OperatorHub is now located under the *Operators* menu.
+* There is now support for chargeback. You can break down the reserved and used resources requested by applications.
+* There is now support for native templates.
+
+[id="ocp-4-2-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} 4.2 introduces the following notable technical changes.
+
+[discrete]
+[id="ocp-4-2-layers"]
+==== Builds maintain their layers
+
+In {product-title} 4.2, builds keep their layers by default.
+
+See
+xref:../builds/running-entitled-builds.adoc#builds-strategy-docker-squash-layers_running-entitled-builds[Running
+entitled builds] for more information.
+
+Ingress controller TLS 1.0 and 1.1 support is now disabled to match the
+Mozilla intermediate security profile.
+
+New and upgraded ingress controllers will no longer support these TLS versions.
+
+[discrete]
+[id="ocp-4-2-windows-builds-not-supported"]
+==== Builds on Windows
+
+Builds on Windows nodes are not supported.
+
+[id="ocp-41-deprecated-features"]
+=== Deprecated features
+
+[discrete]
+[id="ocp-4-2-deprecation-of-service-catalog"]
+
+In {product-title} 4.2, the Service Catalog, the Template Service Broker, the
+Ansible Service Broker, and their Operators are deprecated. They will be removed
+in a future {product-title} release.
+
+[discrete]
+[id="ocp-4-2-deprecation-of-oapi-endpoint"]
+==== Deprecation of `/oapi` endpoint from `oc`
+
+The usage of the `/oapi` endpoint from `oc` is being deprecated and will be
+removed in a future release. The `/oapi` endpoint was responsible for serving
+non-group {product-title} APIs and was removed in 4.1.
+
+[discrete]
+[id="ocp-4-2-deprecation-of-short-flag"]
+==== Deprecation of the `-short` flag of `oc version`
+
+The `oc version --short` flag is now deprecated. The `--short` flag printed
+default output.
+
+[discrete]
+[id="ocp-4-2-api-removal"]
+==== Removal of APIs
+
+In a future release, these APIs will be removed:
+
+* operatorsource.operators.coreos.com/v1
+* *.servicecatalog.k8s.io/v1beta1
+* *.automationbroker.io/v1alpha1
+
+[discrete]
+[id="ocp-4-2-oc-adm-migrate-commands-deprecated"]
+==== `oc adm migrate` commands
+
+The `oc adm migrate` command and all of its subcommands are now deprecated.
```

Comment:
@ahardin-rh please update this message to say all subcommands except for `oc adm migrate template-instances`